jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/common/Snackbar.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.jdi.light.angular.elements.common;
```

Comment:
It seems the Snackbar is not a common element if it is composed of several other elements. I think it should be a complex element.

Reply:
fixed